src/main/tools/build-runfiles-windows.cc

```diff
@@ -381,7 +380,7 @@ int wmain(int argc, wchar_t** argv) {
 
   if (argc != 2) {
     fprintf(stderr, "usage: [--allow_relative] [--use_metadata] "
-            "INPUT RUNFILES\n");
+            "manifest_file runfiles_base_dir\n");
```

Comment:
Thanks! The convention in manpages is to put mandatory arguments in angle brackets, e.g. `<manifest_file>`. I think doing that here would increase readability.